autoload/crystalline/theme/shadesofpurple.vim

```diff
@@ -0,0 +1,18 @@
+function! crystalline#theme#shadesofpurple#set_theme() abort
+  call crystalline#generate_theme({
+        \ 'NormalMode':  [[140, 234], ['#A599E9', '#1E1E3F']],
+        \ 'InsertMode':  [[234, 10],  ['#1E1E3F', '#00FF00']],
+        \ 'VisualMode':  [[234, 177], ['#1E1E3F', '#c991ff']],
+        \ 'ReplaceMode': [[234, 204], ['#1E1E3F', '#FF628C']],
+        \ '':            [[234, 140], ['#1E1E3F', '#A599E9']],
+        \ 'Inactive':    [[234, 140], ['#1E1E3F', '#A599E9']],
+        \ 'Fill':        [[140, 234], ['#A599E9', '#1E1E3F']],
+        \ 'Tab':         [[234, 140], ['#1E1E3F', '#A599E9']],
+        \ 'TabType':     [[234, 140], ['#1E1E3F', '#A599E9']],
+        \ 'TabSel':      [[234, 140], ['#1E1E3F', '#A599E9']],
```

Comment:
You probably want to make `TabSel` distinct from `Tab`, so if mode colors are diabled you can still see the selected tab.